test/FunctionalTestUtils/BackTelemetryChannel.cs

```diff
@@ -37,12 +37,12 @@ public string EndpointAddress
         {
             get
             {
-                throw new NotImplementedException();
```

Comment:
create an issue on server-dotnet repo to not fail dependency collection when channel misbehaving